Pipfile

```diff
@@ -1,10 +1,10 @@
 [packages]
 requests_toolbelt = "*"
+yarl = "*"
 
 [dev-packages]
 oneidentity-safeguard-sessions-plugin-sdk = "*"
 
 [requires]
-python_version = "3.6"
-
+python_version = "3.10"
```

Comment:
This should match the version of python on SPS

Reply:
Same as above

Reply:
on SPS 7.4.0 the python version is 3.10.x
so is it wrong?